src/scenes/AcceleratorRouter/CohortView.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useCallback, useEffect, useMemo } from 'react';
+import { useParams } from 'react-router-dom';
+import { useHistory } from 'react-router';
+import { Grid } from '@mui/material';
+import strings from 'src/strings';
+import { APP_PATHS } from 'src/constants';
+import { useLocalization } from 'src/providers';
+import useStateLocation, { getLocation } from 'src/utils/useStateLocation';
+import { useAppDispatch, useAppSelector } from 'src/redux/store';
+import { selectCohort } from 'src/redux/features/cohorts/cohortsSelectors';
+import { requestCohort } from 'src/redux/features/cohorts/cohortsAsyncThunks';
+import Card from 'src/components/common/Card';
+import Button from 'src/components/common/button/Button';
+import { Crumb, Page } from 'src/components/BreadCrumbs';
+import TextField from 'src/components/common/Textfield/Textfield';
+
+const CohortView = () => {
+  const dispatch = useAppDispatch();
+  const history = useHistory();
+  const location = useStateLocation();
+  const { activeLocale } = useLocalization();
+  const pathParams = useParams<{ cohortId: string }>();
+  const cohortId = Number(pathParams.cohortId);
+
+  const cohort = useAppSelector(selectCohort(cohortId));
+
+  useEffect(() => {
+    void dispatch(requestCohort({ cohortId }));
+  }, [cohortId, dispatch]);
+
+  const goToEditCohort = useCallback(
+    () =>
+      history.push(getLocation(APP_PATHS.ACCELERATOR_COHORTS_EDIT.replace(':cohortId', pathParams.cohortId), location)),
+    [history, location, pathParams.cohortId]
+  );
+
+  const rightComponent = useMemo(
+    () =>
+      activeLocale && (
+        <Button label={strings.EDIT_COHORT} icon='iconEdit' onClick={goToEditCohort} size='medium' id='editCohort' />
+      ),
+    [goToEditCohort, activeLocale]
+  );
+
+  const crumbs: Crumb[] = useMemo(
+    () => [
+      {
+        name: activeLocale ? strings.COHORTS : '',
+        to: APP_PATHS.ACCELERATOR_COHORTS,
+      },
+    ],
+    [activeLocale]
+  );
+
+  return (
+    <Page crumbs={crumbs} title={cohort?.name || ''} rightComponent={rightComponent}>
```

Comment:
Oh wow, didn't realize we had a Page component.. I created another one recently 🤦

Reply:
Hehe yea I feel like there are a few floating around...